controller/cmd/service-mirror/main.go

```diff
@@ -44,11 +69,17 @@ func Main(args []string) {
 		log.Fatalf("Failed to initialize K8s API: %s", err)
 	}
 
+	secretsInformer, err := initLocalSecretsInformer(k8sAPI.Client, *namespace)
+
+	if err != nil {
+		log.Fatalf("Failed to initialize secrets informer: %s", err)
+	}
+
 	probeManager := NewProbeManager(k8sAPI)
 	probeManager.Start()
 
 	k8sAPI.Sync(nil)
```

Comment:
TIOLI: we could add a TODO here for adding a proactive `can-i` check to catch RBAC errors here that would not be caught by `linkerd check` in private clusters.